fedcloud_vm_monitoring/sla_monitor_cli.py

```diff
@@ -0,0 +1,84 @@
+"""Monitor Accounting status"""
+
+import importlib
+
+import click
+import yaml
+from fedcloud_vm_monitoring.accounting import Accounting
+from fedcloud_vm_monitoring.appdb import AppDB
+from fedcloud_vm_monitoring.goc import GOCDB
+
+
+def check_site_slas(site, site_slas, goc, acct, appdb):
+    click.echo(f"[-] Checking site {site}")
+    sla_vos = set()
+    appdb_vos = set(appdb.get_vo_for_site(site))
+    if site not in site_slas:
+        click.echo(f"[I] {site} is not present in any SLA")
+    else:
+        for sla_name, sla in site_slas[site].items():
+            sla_vos = sla_vos.union(sla["vos"])
+            accounted_vos = sla["vos"].intersection(acct.site_vos(site))
+            if accounted_vos:
+                click.echo(
+                    f"[OK] SITE {site} has accouting info for SLA {sla_name} ({accounted_vos})"
```

Comment:
```suggestion
                    f"[OK] SITE {site} has accounting info for SLA {sla_name} ({accounted_vos})"
```

Reply:
I don't know how to write accounting :) 
It reminds me of a writing on a wall near home:
`Los disléxicos también somos persianas`